### PR TITLE
Des 127 update UI backend and db to receive and display new data

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

File: routes/auth.js [hunk 1]
```diff
@@ -23,6 +23,10 @@ router.post(
         check("password", "El password debe de ser de 6 caracteres").isLength({
             min: 6,
         }),
+        check("lastname", "El apellido  es obligatorio").not().isEmpty(),
+        check("country", "La pais es obligatorio").not().isEmpty(),
```
Comment (DanielCalleCO) [inline routes/auth.js:27]:
```suggestion
        check("country", "El país es obligatorio.").not().isEmpty(),
```

File: routes/auth.js [hunk 2]
```diff
@@ -23,6 +23,10 @@ router.post(
         check("password", "El password debe de ser de 6 caracteres").isLength({
             min: 6,
         }),
+        check("lastname", "El apellido  es obligatorio").not().isEmpty(),
```
Comment (DanielCalleCO) [inline routes/auth.js:26]:
```suggestion
        check("lastname", "El apellido  es obligatorio.").not().isEmpty(),
```

File: routes/auth.js [hunk 3]
```diff
@@ -23,6 +23,10 @@ router.post(
         check("password", "El password debe de ser de 6 caracteres").isLength({
```
Comment (DanielCalleCO) [inline routes/auth.js:23]:
```suggestion
        check("password", "La contraseña debe tener mínimo 6 caracteres.").isLength({
```

File: routes/auth.js [hunk 4]
```diff
@@ -23,6 +23,10 @@ router.post(
...
             min: 6,
         }),
+        check("lastname", "El apellido  es obligatorio").not().isEmpty(),
+        check("country", "La pais es obligatorio").not().isEmpty(),
+        check("city", "La cuidad es obligatorio").not().isEmpty(),
```
Comment (DanielCalleCO) [inline routes/auth.js:28]:
```suggestion
        check("city", "La cuidad es obligatoria.").not().isEmpty(),
```

File: routes/auth.js [hunk 5]
```diff
@@ -23,6 +23,10 @@ router.post(
...
+        check("phone", "La cuidad es obligatorio").not().isEmpty(),
```
Comment (DanielCalleCO) [inline routes/auth.js:29]:
```suggestion
        check("phone", "El teléfono es obligatorio.").not().isEmpty(),
```

File: routes/auth.js
Comment (DanielCalleCO) [inline routes/auth.js:1]: `21    check("name", "El nombre es obligatorio.").not().isEmpty(),`
`22    check("email", "El correo electrónico es obligatorio.").isEmail(),`

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]
Just type changes. Awesome job Dani!

File: routes/auth.js (hunk 3 above)
Comment (Daniielmayo) [reply routes/auth.js:23]: fixed

File: routes/auth.js (hunk 4 above)
Comment (Daniielmayo) [reply routes/auth.js:28]: fixed

File: routes/auth.js (hunk 1 above)
Comment (Daniielmayo) [reply routes/auth.js:27]: fixed

File: routes/auth.js (hunk 2 above)
Comment (Daniielmayo) [reply routes/auth.js:26]: fixed

File: routes/auth.js (hunk 5 above)
Comment (Daniielmayo) [reply routes/auth.js:29]: fixed

File: routes/auth.js
Comment (Daniielmayo) [reply routes/auth.js:1]: fixed

File: routes/auth.js (hunk 1 above)
Comment (DanielCalleCO) [reply routes/auth.js]: Dani, "El país" not "La país"

**Review by DanielCalleCO — APPROVED** [review]

**Review by abdulflorez — APPROVED** [review]
Great job!

**Pull request merged**